src/GitVersionTfsTask/task.json

```diff
@@ -42,13 +51,22 @@
       "required": false,
       "helpMarkDown": "Additional arguments to send to GitVersion.exe",
       "groupName": "additional"
+    },
+    {
+      "name": "gitVersionPath",
```

Comment:
To be honest I was copying something else I'd done before and thought I'd leave it to allow me to test a future version of GitVersion without having to redeploy VSO as it might disrupt the rest of my teams doing builds.